documentation/tut/toy-example-from-scratch.rst

```diff
@@ -3,23 +3,14 @@
 Toy example: Scheduling a battery, from scratch
 ===============================================
 
-Let's walk through an example from scratch! We'll ... 
-
-- install FlexMeasures
-- create an account with a battery asset
-- load hourly prices
-- optimize a 12h-schedule for a battery that is half full
+Let's walk through an example from scratch! We'll optimize a 12h-schedule for a battery that is half full.
 
 What do you need? Your own computer, with one of two situations: either you have `Docker <https://www.docker.com/>`_ or your computer supports Python 3.8+, pip and PostgresDB. The former might be easier, see the installation step below. But you choose.
```

Comment:
This sentence is also in the setup part. I would replace it with a short sentence like "If you haven't run through [Toy example: Introduction and setup] yet, do that first. There, we added a battery asset and variable power prices (for a 24h window)."